src/rosdep2/platforms/debian.py

```diff
@@ -54,6 +55,13 @@ def register_debian(context):
     context.set_default_os_installer_key(OS_DEBIAN, APT_INSTALLER)
     context.set_os_version_type(OS_DEBIAN, OsDetect.get_codename)
     
+def register_linaro(context):
+    # Linaro is an alias for Ubuntu. If linaro is detected and it's not set as an override force ubuntu.
+    (os_name, os_version) = context.get_os_name_and_version()
+    if os_name == OS_LINARO and not context.os_override:
+        print("rosdep detected OS: [%s] aliasing it to: [%s]" % (OS_LINARO, OS_UBUNTU))
```

Comment:
If it always prints, I would at least put it to stderr. I agree that it should be visible though.